### ISSUE TITLE
Switching between profiles

### ISSUE THREAD

**Issue by jbarry786** [stax76/mpv.net#96]
Is it possible or would it be possible to switch between profiles when you already have a video open?

Not sure if I may have been missing something, but i've not seen a way to do with with mpv before and not sure if it's somewhere on mpvnet.

But basically,

When you open a video, have the option of,

Right click-> brings up -->Menu --->Profiles ---> profile 1 / profile 2 / etc

If this is not possible, how about an option where we can enable mpvnet to ask the user what profile to use before playing the file?

**Comment by stax76** [comment]
Maybe customize the menu in input.conf:

_   set profile "profile a"   #menu: Profiles > profile a

https://mpv.io/manual/master/#command-interface-set-%3Cname%3E-%3Cvalue%3E

**Comment by jbarry786** [comment]
Stax, thanks for the reply. For some reason I thought this was actually more complicated than it is, based on me having read [this thread](https://github.com/mpv-player/mpv/issues/4843). Maybe I misunderstood it.

In any case, following your reply, it worked.

On mpv.conf:
`[testprofile]`
`osc=no`

On input.conf:
`_ set profile testprofile #menu: Profiles > profile testprofile`

The context menu reflects those changes.
![image](https://user-images.githubusercontent.com/30951795/65311785-060c9080-db89-11e9-923f-428e89ef928f.png)

The one peculiar thing is that while the profile is changed, on the OSD, you get this message "error):
![image](https://user-images.githubusercontent.com/30951795/65325157-00717380-dba6-11e9-9246-8d6a814e0e81.png)

But the profile did change.

However, I haven't found a way to switch back to a 'default' version. The goal would be to have different profiles and an option to go back to the general settings.
` #menu: Profiles > profile a/profile b/no profile (default/global settings)`

I have tried using `default` but doesn't seem to work. 

I have also tried to set up a key to `cycle profile` but it gives me an error,  "failed to set property profile". While I have been able to set up a profile and assign a key to it. I have not been able to set up a cycle or a key to go back to a different profile once I make a switch.

I might be missing something in what I have been attempting to do and in any case, this is something to do with mpv itself. Thanks for pointing me in the right direction though, enabling me to quickly change to a different profile is what I was looking for.

By the way Stax, when I ran this command `mpvnet --profile=help`  I got this error (it still listed the profiles though):

 Exception
profile: MPV_ERROR_PROPERTY_ERROR
System.Exception: profile: MPV_ERROR_PROPERTY_ERROR
   em mpvnet.mp.set_property_string(String name, String value, Boolean throwOnException) em D:\Projekte\CS\mpv.net\mpv.net\mpv\mp.cs:line 487
   em mpvnet.mp.ProcessCommandLine(Boolean preInit) em D:\Projekte\CS\mpv.net\mpv.net\mpv\mp.cs:line 612

[I am running: mpv.net-portable-x64-5.4.1.0]
